src/authentication/protected-routes.handler.ts

```diff
@@ -10,31 +10,20 @@ export const withProtectedRoutesHandler = (
   const { rootPath, loginPath, logoutPath } = admin.options;
 
   router.use((req, res, next) => {
-    if (isAdminAsset(req.originalUrl)) {
-      next();
-    } else if (
-      req.session.adminUser ||
-      // these routes doesn't need authentication
+    if (
+      // these routes don't need authentication
+      isAdminAsset(req.originalUrl) ||
       req.originalUrl.startsWith(loginPath) ||
       req.originalUrl.startsWith(logoutPath)
     ) {
-      next();
-    } else if (isAdminRoute(req.originalUrl, rootPath)) {
-      // If the redirection is caused by API call to some action just redirect to resource
-      const [redirectTo] = req.originalUrl.split("/actions");
```

Comment:
I'm not sure why it was here previously tbh, I've tested locally and this doesn't seem to be required unless I'm missing something